src/js/hca/config/form.js

```diff
@@ -119,14 +118,13 @@ const formConfig = {
   formId: '1010ez',
   version: 0,
   // Disable save in progress for production
-  disableSave: __BUILDTYPE__ === 'production',
   savedFormErrorMessages: {
     notFound: 'Please start over to apply for health care.',
     noAuth: 'Please sign in again to resume your application for health care.'
   },
   transformForSubmit: transform,
   // Use the old intro page for production, but SiP for dev and staging
-  introduction: __BUILDTYPE__ === 'production' ? IntroductionPage : SIPIntroductionPage,
+  introduction: SIPIntroductionPage,
```

Comment:
I could move SIPIntroductionPage to IntroductionPage, but that would invite merge conflicts.